app/scripts/models/application-settings.coffee

```diff
@@ -399,40 +305,36 @@ define [
 
     defaults: ->
 
-      server1 =
-        id: @guid()
-        name: FieldDB.FieldDBObject.application.brand + ' Localhost'
-        type: 'FieldDB'
-        url: 'https://localhost:3183'
-        serverCode: 'localhost'
-        corpusServerURL: null
```

Comment:
This code is equivalent to what you had.

 Basically, we moved the code that knows the usual server combinations, and knows how to guess server combinations into the commonjs so you can use it. I also incorporated some of your feedback about offline replication urls and logging the user into their corpus server when they login to the auth service.
https://github.com/OpenSourceFieldlinguistics/FieldDB/blob/master/api/corpus/Connection.js#L560-570

This shows how it works: https://github.com/OpenSourceFieldlinguistics/FieldDB/blob/master/tests/corpus/DatabaseTest.js#L438-468

Here is an example of how its used:
https://github.com/OpenSourceFieldlinguistics/AuthenticationWebService/blob/master/lib/nodeconfig_local.js#L19-37